src/main/java/PeselValidator.java

```diff
@@ -0,0 +1,134 @@
+
+public class PeselValidator {
+
+
+    public boolean validate(String pesel) {
+
+        if (!checkPeselLength(pesel)) {
```

Comment:
It's very good idea to split the functionality to smaller functions, but even better you could use short circuit operator to write this code in one line:
```suggestion
        return checkPeselLength(pesel) && checkPeselMonth(pesel) && checkPeselDay(pesel) && checkPeselLastNumber(pesel);
```